modules/gitops-argo-cd-properties.adoc

```diff
@@ -11,19 +11,28 @@ The Argo CD Custom Resource consists of the following properties:
 [options="header"]
 |===
 |Name |Description |Default |Properties
+
 |`applicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance` |
 
-|`applicationSet`
-|`applicationSet` controller configuration options.
-| `_<object>_`
-a|* `<image>` - The container image for the `applicationSet` controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
+|`aggregatedClusterRoles`    |The aggregation of several cluster roles for a cluster-scoped instance. | `__<empty>__` |
+
+|`applicationSet` |The ApplicationSet Controller configuration options. | `_<object>_`
```

Comment:
```suggestion
|`applicationSet` |The ApplicationSet controller configuration options. | `_<object>_`
```
In the Argo CD documentation, I generally see "controller" lowercase in the phrase "ApplicationSet controller".

Reply:
Hi @lahinson, in the gitops documentation under the section [Glossary of common terms for OpenShift GitOps](https://81661--ocpdocs-pr.netlify.app/openshift-gitops/latest/understanding_openshift_gitops/about-redhat-openshift-gitops#gitops-openshift-go-common-terms_about-redhat-openshift-gitops) we've used capital C for Application Controller so I would like to stick to it to maintain consistency. WDYT?

Reply:
Thanks for letting me know. Using the capital C to maintain consistency sounds reasonable.